src/app/widgets/inlineinput/inlineinput.component.ts

```diff
@@ -76,12 +86,24 @@ export class InlineInputComponent implements OnInit {
     }
   }
 
+  validateValue(value) {
+    if (this.type === 'integer') {
+      return /^\d+$/.test(value);
+    }
+    if (this.type === 'float') {
+      return /^-?\d*(\.\d+)?$/.test(value);
```

Comment:
A float can never have negative value because of this https://github.com/fabric8-ui/fabric8-planner/pull/2680/commits/43ab0dc0d9a4686db49bfe559adc924d728c080a#diff-ab69836b288ac3d161b883556bbbebc3R171 function. The input doesn't accept `-` character.